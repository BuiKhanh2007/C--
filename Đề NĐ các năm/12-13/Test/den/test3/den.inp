117 212
64 197
162 201
196 397
17 42
179 252
