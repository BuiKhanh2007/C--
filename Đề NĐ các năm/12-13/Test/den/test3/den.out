1183
1525
502
2979
245
1037
